package.json

```diff
@@ -43,8 +43,8 @@
     "@testing-library/react-hooks": "^7.0.2",
     "@types/jest": "^29.0.3",
     "@types/node": "^18.7.18",
-    "@types/react": "^17.0.38",
-    "@types/react-dom": ">=16.9.0",
+    "@types/react": "^18.0.0",
+    "@types/react-dom": "^18.0.0",
```

Comment:
This works when used by react@18 projects, but it doesn't work else on my tests on a Gatsby powered react@17 website.